components/ws-manager/pkg/manager/testdata/status_headlessTaskFailed_STOPPING00.json

```diff
@@ -0,0 +1,1223 @@
+                "f:cni.projectcalico.org/podIP": {},
+                "f:cni.projectcalico.org/podIPs": {}
+              }
+            }
+          }
+        },
+        {
+          "manager": "ws-manager",
+          "operation": "Update",
+          "apiVersion": "v1",
+          "time": "2021-07-27T18:29:06Z",
+          "fieldsType": "FieldsV1",
+          "fieldsV1": {
+            "f:metadata": {
+              "f:annotations": {
+                ".": {},
+                "f:cluster-autoscaler.kubernetes.io/safe-to-evict": {},
+                "f:container.apparmor.security.beta.kubernetes.io/workspace": {},
+                "f:gitpod.io/requiredNodeServices": {},
+                "f:gitpod/admission": {},
+                "f:gitpod/contentInitializer": {},
+                "f:gitpod/id": {},
+                "f:gitpod/imageSpec": {},
+                "f:gitpod/never-ready": {},
+                "f:gitpod/ownerToken": {},
+                "f:gitpod/servicePrefix": {},
+                "f:gitpod/traceid": {},
+                "f:gitpod/url": {},
+                "f:prometheus.io/path": {},
+                "f:prometheus.io/port": {},
+                "f:prometheus.io/scrape": {},
+                "f:seccomp.security.alpha.kubernetes.io/pod": {}
+              },
+              "f:labels": {
+                ".": {},
+                "f:app": {},
+                "f:component": {},
+                "f:gitpod.io/networkpolicy": {},
+                "f:gpwsman": {},
+                "f:headless": {},
+                "f:metaID": {},
+                "f:owner": {},
+                "f:workspaceID": {},
+                "f:workspaceType": {}
+              }
+            },
+            "f:spec": {
+              "f:affinity": {
+                ".": {},
+                "f:nodeAffinity": {
+                  ".": {},
+                  "f:requiredDuringSchedulingIgnoredDuringExecution": {
+                    ".": {},
+                    "f:nodeSelectorTerms": {}
+                  }
+                }
+              },
+              "f:automountServiceAccountToken": {},
+              "f:containers": {
+                "k:{\"name\":\"workspace\"}": {
+                  ".": {},
+                  "f:command": {},
+                  "f:env": {
+                    ".": {},
+                    "k:{\"name\":\"GITPOD_CLI_APITOKEN\"}": {
+                      ".": {},
+                      "f:name": {},
+                      "f:value": {}
+                    },
+                    "k:{\"name\":\"GITPOD_EXTERNAL_EXTENSIONS\"}": {
+                      ".": {},
+                      "f:name": {},
+                      "f:value": {}
+                    },
+                    "k:{\"name\":\"GITPOD_GIT_USER_EMAIL\"}": {
+                      ".": {},
+                      "f:name": {},
+                      "f:value": {}
+                    },
+                    "k:{\"name\":\"GITPOD_GIT_USER_NAME\"}": {
```

Comment:
Can we somehow get rid of this inline schemas in test data? :thinking:

Reply:
or in Kubernetes in general :D 

I'm not sure what the effect would be if we removed this. Probably none.